config/locales/client.en.yml

```diff
@@ -1928,6 +1942,8 @@ en:
     posts: "Posts"
     posts_lowercase: "posts"
     posts_long: "there are {{number}} posts in this topic"
+
+    # keys ending with _MF use message format, see /spec/components/js_local_helper_spec.rb for samples
```

Comment:
I guess it would be a good idea to link to https://meta.discourse.org/t/message-format-support-for-localization/7035 instead. Maybe update the existing comment that you copied too. The topic on meta is a lot more helpful than the spec.